llm_on_ray/inference/api_openai_backend/router_app.py

```diff
@@ -68,7 +68,7 @@
 logger = get_logger(__name__)
 
 # timeout in 10 minutes. Streaming can take longer than 3 min
-TIMEOUT = float(os.environ.get("ROUTER_HTTP_TIMEOUT", 600))
+TIMEOUT = float(os.environ.get("ROUTER_HTTP_TIMEOUT", 600000000000000))
```

Comment:
This is because when sending 1000 prompts, this time will be exceeded, causing interruption.